packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -70,6 +70,75 @@ const DEFAULT_SCHEDULE = {
   timezone: moment.tz.guess(),
 }
 
+const ReportReceivers = decorate([
+  provideState({
+    initialState: () => ({
+      receiver: '',
+    }),
+    effects: {
+      linkState,
+      addReceiver() {
+        const { receiver } = this.state
+        if (receiver === '') {
+          return
+        }
+        this.props.addReceiver(receiver)
+        this.resetState()
+      },
+      onKeyDown({ addReceiver }, event) {
+        if (event.keyCode === 13) {
```

Comment:
what's this magic number ? I think it should be defined in a `const`

Reply:
It's not a number defined by me. It's a [keyboard event code](https://lists.w3.org/Archives/Public/www-dom/2010JulSep/att-0182/keyCode-spec.html).